Gems/Multiplayer/Code/Include/Multiplayer/Components/NetworkCharacterComponent.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Source/AutoGen/NetworkCharacterComponent.AutoComponent.h>
+#include <PhysX/CharacterGameplayBus.h>
+#include <Multiplayer/Components/NetBindComponent.h>
+
+namespace Physics
+{
+    class Character;
+}
+
+namespace Multiplayer
+{
+    class NetworkCharacterComponent
```

Comment:
This makes me wonder if we should have a Description element in the AutoComponent xml that renders to the ComponentBase. That feels more consistent with how AutoComponents are otherwise documented. If so, that should be addressed in a separate PR.

Reply:
fixed!